src/parsers/power_system_table_data.jl

```diff
@@ -936,6 +958,17 @@ function make_reactive_params(
     return reactive_power, reactive_power_limits
 end
 
+function add_outage_info!(component::StaticInjection, gen)
+    outage_rates = outage_to_rate((gen.fotr, gen.mttr))
+    outage_probability = outage_info(outage_rates.λ, outage_rates.μ)
+
+    component.ext = Dict(
```

Comment:
Using `ext` in this way could be very problematic because it will discard any existing data. Instead, you should do this:
```
ext = get_ext(component)
for fn in fieldnames(outage_info)
    ext[string(fn)] = getfield(outage_probability, fn)
end
```

Reply:
Yeah, makes sense. The new commit includes these changes.